README.md

```diff
@@ -142,9 +154,10 @@ Discussion should be held in the open whenever possible.  However, if you need t
 
 ### Project Resources & Travel Fund
 
-OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md).
+OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.
+
+OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration 
```

Comment:
This sentence doesn't seem to be complete. I am not entirely sure which section it refers to.

Reply:
@christian-bromann thanks for catching that. Fixed although the link won't work until https://github.com/openjs-foundation/cross-project-council/pull/595 lands